raiden/tasks.py

```diff
@@ -95,6 +98,27 @@ def check_gas_reserve(raiden):
         gevent.sleep(CHECK_GAS_RESERVE_INTERVAL)
 
 
+def check_rdn_deposits(raiden, user_deposit_proxy: UserDeposit):
+    """ Check periodically for RDN deposits in the user-deposits contract """
+    while True:
+        rdn_balance = user_deposit_proxy.effective_balance(raiden.address)
+
+        if rdn_balance < MIN_RDN_THRESHOLD:
+            click.secho(
+                (
+                    'WARNING\n'
+                    f"Your account's RDN balance of {rdn_balance} is below the "
+                    'minimum threshold. Provided that you have either a monitoring '
+                    'service or a path finding service activated, your node is not going'
+                    'to be able to pay those services which may lead to denial of service or '
+                    'loss of funds.'
+                ),
+                fg='red',
```

Comment:
Also note. Not for this PR but we really need to discuss how we handle all these warnings that need user action. This  print statement is effectively useless as the user will probably not even see it.

Reply:
I believe the webUI should show these warnings... should we create an issue to return warnings to the webUI via an API call?

Reply:
Yeah that's a good idea. cc @kelsos

Reply:
Created: https://github.com/raiden-network/raiden/issues/3631